config/initializers/filter_parameter_logging.rb

```diff
@@ -4,5 +4,5 @@
 # sensitive information. See the ActiveSupport::ParameterFilter documentation for supported
 # notations and behaviors.
 Rails.application.config.filter_parameters += [
-  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn, :email, :name, :username, :password, :authenticity_token
+  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn, :email, :name, :username, :password, :authenticity_token, :access_key, :secret_access_key
```

Comment:
Probably leave those in there just in case, we could also add the params being sent around for new/previous owner emails